docs/reference/mpremote.rst

```diff
@@ -0,0 +1,90 @@
+.. _mpremote_files:
+
+MicroPython .mpremote files
+===========================
+
+This CLI tool provides an integrated set of utilities to remotely interact with and automate a MicroPython device over a serial connection.
+
+The simplest way to use this tool is:
+
+mpremote
+
+This will automatically connect to the device and provide an interactive REPL.
+
+The full list of supported commands are:
+
+* mpremote <device-shortcut>       -- connect to the given device
+
+* mpremote connect <device>        -- connect to given device
+                                    device may be: list, auto, id:x, port:x
+                                    or any valid device name/path
+
+* mpremote disconnect              -- disconnect current device
+
+* mpremote mount <local-dir>       -- mount local directory on device
+
+* mpremote eval <string>           -- evaluate and print the string
+
+* mpremote exec <string>           -- execute the string
+
+* mpremote run <file>              -- run the given local script
+
+* mpremote fs <command> <args...>  -- execute filesystem commands on the device
+                                    command may be: cat, ls, cp, rm, mkdir, rmdir
+                                    use ":" as a prefix to specify a file on the device
+
+* mpremote repl                    -- enter REPL
+  
+                                    options:
+                                        --capture <file>
+                                        --inject-code <string>
+                                        --inject-file <file>
```

Comment:
Need to find a nice way to format this section. Maybe switch the description and usage around, eg:
```
* Connect to the given device:

    mpremote connect <device>
```